src/pages/vault/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { GetStaticPathsResult } from 'next'
+import vaultData from 'data/vault.json'
+import { ProductOption } from 'lib/learn-client/types'
+import TutorialView from 'views/tutorial-view'
+import {
+  getTutorialPagePaths,
+  getTutorialPageProps,
+  TutorialPageProps,
+  TutorialPagePaths,
+  TutorialPageProduct,
+} from 'views/tutorial-view/server'
+import CoreDevDotLayout from 'layouts/core-dev-dot-layout'
+
+export function VaultTutorialPage({
+  tutorial,
+  layoutProps,
+}: TutorialPageProps): React.ReactElement {
+  return <TutorialView tutorial={tutorial} layout={layoutProps} />
+}
+
+export async function getStaticProps({
+  params,
+}): Promise<{ props: TutorialPageProps }> {
+  const product = {
+    slug: vaultData.slug,
+    name: vaultData.name,
+  } as TutorialPageProduct
```

Comment:
I'm considering refactoring this to source product data from the Learn API instead. Anyone have thoughts? The product hub page already does this. I want to maintain consistency with the Learn data that authors can edit. I'm sure the values are the same. 

I won't make these changes in this PR, but I did spike out the [updates here](https://github.com/hashicorp/dev-portal/pull/197) for reference on what i'm thinking

Reply:
I'd be on board with this 👍 I can see the case for wanting to get product data from a single source, and but as long as we still have the Learn API source and are using it elsewhere, I think it makes sense to match This Learn consumer with the Learn source 🤝 

(I think you had a similar comment on another PR so apologies if I'm being repetitive 🙇‍♂️)

Reply:
I agree that we need 1 source of truth in the long term, and I also think we need a more ergonomic way to hydrate product data for various parts of the app, but I don't think we're ready for it yet. One major thing that's missing from these tutorial and collection pages is the links in the main navigation header, and that data is sourced from the `src/data/{productSlug}.json` JSON files. There should be more data used from those files here in addition to `slug` and `name`, it just hasn't been implemented yet. So right now it probably looks like the data files have little impact and can be easily refactored away from. There will be some changes to the navigation component's structure over the next few weeks, but the data is still going to be stored in the same JSON file for the time being.

Reply:
I think it makes sense to source nav data from a central place (the static JSON files). But for the product data I'm leaning towards sourcing that from the Learn API.